File: threedollar-boss-domain/src/main/java/com/depromeet/threedollar/foodtruck/domain/domain/account/BossAccountRepository.java
package com.depromeet.threedollar.foodtruck.domain.domain.account;

import com.depromeet.threedollar.foodtruck.domain.domain.account.repository.BossAccountRepositoryCustom;
import org.springframework.data.jpa.repository.JpaRepository;

public interface BossAccountRepository extends JpaRepository<BossAccount, Long>, BossAccountRepositoryCustom {

}

File: threedollar-domain/threedollar-domain-mongo/src/main/kotlin/com/depromeet/threedollar/domain/mongo/foodtruck/domain/account/BossAccountRepository.kt
package com.depromeet.threedollar.domain.mongo.foodtruck.domain.account

import org.springframework.data.mongodb.repository.MongoRepository
import com.depromeet.threedollar.domain.mongo.foodtruck.domain.account.repository.BossAccountRepositoryCustom
import com.depromeet.threedollar.domain.mongo.foodtruck.domain.account.repository.BossAccountStatisticsRepositoryCustom

interface BossAccountRepository : MongoRepository<BossAccount, String>, BossAccountRepositoryCustom, BossAccountStatisticsRepositoryCustom
